components/molecule/pagination/src/index.scss

```diff
@@ -8,6 +9,8 @@ $trf-pagination-next-button-icon: none !default;
 $c-pagination-button-icon-path: inherit !default;
 
 .sui-MoleculePagination {
+  @include sui-list('inline');
```

Comment:
Since this is an utility that you can develop with 2 or 3 lines of css, I would avoid adding a `settings-compat-v7` folder. Actually, we should tend to eliminate this folder.

Reply:
Ok 😢 😅

Reply:
This folder was meant to maintain compatibility with the first components we made. so hopefully we'll delete soon, and we'll avoid incorrect use of it. @turolopezsanabria